### DISCUSSION TITLE
File Explorer?

### DISCUSSION THREAD

**Discussion by josephcrawfordSRH** [johnlindquist/kit#1380] [Q&A] [answered]
I am writing a script that will allow someone to select their `projects` directory and then will get all of the projects which are under GIT and cache them.  They will then be used to allow a user to open a project on a specific branch.

Currently I am hard-coding my project directory but if I let the team use this their location is probably different.  Is there a way with an `arg()` or some other function to open a file explorer so they can select their folder and have it just put the path in place?  I want this to support both mac and windows so I don't want to hard code anything if at all possible but also keep them from having to type a really long path.

So far everything is working except for having to have the path hard-coded.

**Comment by johnlindquist** [comment, marked as answer]
@josephcrawfordSRH You can use `path`

let selectedPath = await path()

**Reply by josephcrawfordSRH** [reply]
perfect thank you!
